### ISSUE TITLE
Option to select molecular profile in alterations tab

### ISSUE THREAD

**Issue by kalletlak** [cBioPortal/cbioportal#8369]
Studies might have multiple molecular profile of same type. Example: https://www.cbioportal.org/study?id=gbm_tcga_pub has 2 discrete CNA profiles. Before we had [dropdown](https://www.cbioportal.org/results/comparison?cancer_study_list=gbm_tcga_pub&tab_index=tab_visualize&case_set_id=gbm_tcga_pub_all&Action=Submit&gene_list=TP53%250APTEN%250ACDKN2A%250ACDKN2B&comparison_subtab=cna) to select which profile it is. With these new alteration changes this selection is lost.

**Old**
![Screen Shot 2021-02-23 at 12 25 43 PM](https://user-images.githubusercontent.com/12956870/108882395-57b51800-75d2-11eb-9b8a-383a1c4e59a1.png)

**New**
![Screen Shot 2021-02-23 at 12 17 13 PM](https://user-images.githubusercontent.com/12956870/108881333-2f78e980-75d1-11eb-83d5-f7fb26bf50d7.png)

**Comment by jjgao** [comment]
Thannks, @kalletlak.

@pvannierop I would be great if we can have a follow-up PR to allow users to select profiles once this feature is merged and released. Not sure what would be a good UI to support that though. cc'ing @alisman.

**Comment by alisman** [comment]
see line 625 of src/shared/lib/comparison/ComparisonStore.ts
where we select first index of array.

check to see if it's even possible to have more than one profile for Mutations or SV
